xml/book_cloud_suppl.xml

```diff
@@ -1406,6 +1406,13 @@
       The <systemitem class="resource">python-magnumclient</systemitem> is installed.
      </para>
     </listitem>
+            <listitem>
+     <para>
+    Install the <package>openstack-magnum-k8s-image-x86_64</package> package. 
+    This provides a Kubernetes-based virtual machine, <filename>openstack-magnum-k8s-image.x86_64.qcow2</filename>, that provides OpenStack Magnum Container 
+    Orchestration.
```

Comment:
This provides a SLES12SP2 based docker image, <filename>openstack-magnum-k8s-image.x86_64.qcow2</filename>, on which you can deploy kubernetes.